plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcPageSink.java

```diff
@@ -61,10 +61,10 @@ public JdbcPageSink(ConnectorSession session, JdbcOutputTableHandle handle, Jdbc
         }
 
         try {
-            // According to JDBC javaodcs "If a connection is in auto-commit mode, then all its SQL statements will be
+            // According to JDBC javadocs "If a connection is in auto-commit mode, then all its SQL statements will be
```

Comment:
Let's fix the commit title as "Fix grammar and typo in JdbcPageSink" if you agree with other comment.